src/js/containers/ActiveZoneSelector/ActiveZoneSelector.js

```diff
@@ -40,16 +40,29 @@ class ActiveZoneSelector extends Component {
     }
 
     render() {
-        let { activeZone, intl, zoneList } = this.props;
+        let { activeZone, zoneList } = this.props;
         let zones = _.values(zoneList).map(zone => {
             return { value: zone.name, label: zone.name };
         });
 
+        let activeZoneName = activeZone.name;
```

Comment:
If we only need the `name` from `activeZone` lets update `mapStateToProps()` to have:

`activeZoneName: state.activeZone.name`